### PR TITLE
firstday 

### PR DESCRIPTION
**Pull request by JeanMichelBB**
I have some difficulty with the button part of the `App.js` file for 'or and choose a file.' It's challenging to position it correctly without the excessive use of `<br />` elements.

### CODE REVIEW

File: index.html
Comment (anyruizd) [inline index.html:1]: I think you can get rid of this file

File: src/App.css [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+}
+
+.Or {
```
Comment (anyruizd) [inline src/App.css]: Avoid the use of capital letters in the class names. Read this for more info https://stackoverflow.com/a/2345300

File: src/App.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
+// App.js
+import React from 'react';
+import MyComponent from './components/MyComponent';
```
Comment (anyruizd) [inline src/App.js]: Why are you importing `MyComponent` if you are not using it?

File: src/App.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+const App = () => {
+  return (
+    <div class="rectangle-container">
+      <input style={{ display: 'none' }} type="file" accept="image/*" />
+      <br />
+      <h2 class="title">Uploading your image</h2>
+      <br />
+      <p class="subtitle">File should be Jpeg, Png,...</p>
+      <div  className="square-container" type="file" accept="image/*" />
```
Comment (anyruizd) [inline src/App.js]: The `div` tag is not self-closing; that's probably why you have to over use `<br />`.

File: src/App.js [hunk 4]
```diff
@@ -0,0 +1,42 @@
+// App.js
+import React from 'react';
+import MyComponent from './components/MyComponent';
+import './App.css';
+
+const App = () => {
+  return (
+    <div class="rectangle-container">
+      <input style={{ display: 'none' }} type="file" accept="image/*" />
+      <br />
```
Comment (anyruizd) [inline src/App.js]: The spacing given by the `br` can be accomplished using display flex and the [gap property](https://css-tricks.com/snippets/css/a-guide-to-flexbox/#aa-gap-row-gap-column-gap); this property can also help you to position all the elements in the container easily.

File: src/App.css [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+    border-radius: 10px;
+    border: 4px dashed rgb(125, 212, 252);
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+}
```
Comment (anyruizd) [inline src/App.css]: I don't think you need to use position absolute AND flex at the same time in this case 🤔

File: src/App.js (hunk 2 above)
Comment (JeanMichelBB) [reply src/App.js]: It was for testing I'm going to use it later but it's true for the moment it's not useful.

File: src/App.js [hunk 6]
```diff
@@ -0,0 +1,42 @@
+// App.js
+import React from 'react';
+import MyComponent from './components/MyComponent';
+import './App.css';
+
+const App = () => {
+  return (
+    <div class="rectangle-container">
```
Comment (anyruizd) [inline src/App.js]: When you're using react, you need to use `className` instead of `class`, because class is a reserved word.

**Pull request merged**